tests/src/test/java/alluxio/shell/command/StatCommandTest.java

```diff
@@ -22,30 +22,51 @@
 import java.io.IOException;
 
 /**
- * Tests for fileInfo command.
+ * Tests for stat command.
```

Comment:
Although `getFileInfo` is deprecated, I would prefer to keep the tests for it and make a new copy for testing `stat`.